components/builder-api-client/src/builder.rs

```diff
@@ -31,8 +30,7 @@ use crate::{error::{Error,
                                  PackageIdent,
                                  PackageTarget},
                        ChannelIdent},
-            hab_http::{util::decoded_response,
-                       ApiClient},
```

Comment:
Same comment here as above re: deleting all the blank lines in between the `use` statements.

Reply:
:+1: